roles/certbot/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+- name: Stop keepalived service on second node
+  systemd:
+    name: keepalived
+    state: stopped
+    enabled: yes
+  when: inventory_hostname == groups.loadbalancers[1]
+  become: yes
+  become_method: sudo
+- name: Enable optional and extras repos
```

Comment:
(optional) a blank line between each task might make it easier to read

Reply:
This is done now.